.github/ISSUE TEMPLATES/node-related-package-releases.md

```diff
@@ -0,0 +1,39 @@
+---
+name: Node-related package releases
+about: Use this template for tracking package releases that pertain to a specific release of `cardano-node`
+title: "Release: for cardano-node [VERSION]"
+---
+
```

Comment:
Could you please add a section here with:

* a script to run in CHaP repo that would attempt release all of the packages from the repo
* check box for creating git tags for released packages
* check box for creating a PR with post release process (add sections to changelogs)